modules/logging/pom.xml

```diff
@@ -24,6 +24,11 @@
       <artifactId>jcl-over-slf4j</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.owasp</groupId>
+      <artifactId>security-logging-logback</artifactId>
```

Comment:
Please declare this dependency as optional.
You may add it (non optional) in current sample app. People who do not want it can then remove it from there.
But logging module should not force you to use logback nor OWASP stuff.